waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -551,10 +553,12 @@ method isReady*(g: OnchainGroupManager): Future[bool] {.async,gcsafe.} =
   if g.ethRpc.isNone():
     return false
 
-  if g.lastSeenBlockHead == 0:
-    return false
+  let currentBlock = cast[BlockNumber](await g.ethRpc
```

Comment:
guess if this fails an exceptions is thrown? should we catch it and return `false` if so. guess if we dont catch it the node will crash?